flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/TaskExecutorResourceUtilsTest.java

```diff
@@ -310,21 +310,6 @@ public void testConfigManagedMemoryFractionFailure() {
 		validateFailInConfigurationsWithoutExplicitManagedMem(conf);
 	}
 
-	@Test
-	public void testConfigManagedMemoryLegacyFraction() {
```

Comment:
a bit strange that we have this commit before main commits and not as a part of e.g.
09bed0858a - [FLINK-13986][core][config] Remove legacy config option TaskManagerOptions#LEGACY_MANAGED_MEMORY_FRACTION